graviti/dataframe/column/series.py

```diff
@@ -225,3 +232,38 @@ def loc(self) -> ColumnSeriesLocIndexer:
 
         """
         return ColumnSeriesLocIndexer(self)
+
+    @classmethod
+    def _from_pyarrow(
+        cls: Type[_T], array: pa.Array, schema: PortexType, name: Optional[str] = None
+    ) -> _T:
+        obj: _T = object.__new__(cls)
+        obj._data = PagingList(array)  # type: ignore[assignment]
```

Comment:
where the mypy error comes from ?

Reply:
In `__init__` function, `Series._data` has the type `Sequence[Any]`

Reply:
shall we update the type of `_data`,
Should the `_data` always be `PagingList`?

Reply:
Maybe we no need update the type now, 
We can update the type after implemented `Series.__init__` with the PagingList.